src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int maxSizeValue = 50;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier(random);
+
+    public Figure getRandomFigure() {
+        switch (Figures.values()[random.nextInt(Figures.values().length)]) {
+            case CIRCLE:
+                return getCircle();
+            case SQUARE:
+                return getSquare();
+            case RECTANGLE:
+                return getRectangle();
+            case RIGHTTRIANGLE:
+                return getRightTriangle();
+            default:
+                return getIsoscelesTrapezoid();
```

Comment:
maybe it would be better to use `public Figure getDefaultFigure()` method for default case